Sources/MongoSwift/Operations/CountDocumentsOperation.swift

```diff
@@ -64,21 +64,28 @@ internal struct CountDocumentsOperation<T: Codable>: Operation {
     internal func execute(using connection: Connection, session: ClientSession?) throws -> Int {
         let opts = try encodeOptions(options: options, session: session)
         var error = bson_error_t()
-        var reply = BSONDocument()
-        let count = self.collection.withMongocCollection(from: connection) { collPtr in
-            self.filter.withBSONPointer { filterPtr in
-                withOptionalBSONPointer(to: opts) { optsPtr in
-                    ReadPreference.withOptionalMongocReadPreference(from: self.options?.readPreference) { rpPtr in
-                        reply.withMutableBSONPointer { replyPtr in
-                            mongoc_collection_count_documents(collPtr, filterPtr, optsPtr, rpPtr, replyPtr, &error)
+
+        return try self.collection.withMongocCollection(from: connection) { collPtr in
```

Comment:
this is actually totally unrelated, but I encountered it when running the manual leak checking via the atExit handler hack described here. the problem is the leak of the reply `bson_t`.
we actually fixed all other occurrences of this problem back in https://github.com/mongodb/mongo-swift-driver/commit/b7c310a1a32c26053b1c3cc4a3d3c025896d30d5 / SWIFT-824; I'm not sure how I missed this one then. 🤔